src/main/java/interface_adapter/chat/ChatRestController.java

```diff
@@ -0,0 +1,67 @@
+package interface_adapter.chat;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
```

Comment:
Clean and concise format. i.e. Use of final keyword where variables are not re-assigned enhances immutability & readability!